app/assets/stylesheets/form.scss

```diff
@@ -1,12 +1,12 @@
-@import "foundation_and_overrides";
-
-$h1-color:      #088da5;
 $border-color:  #000000;
+$h1-color: #088da5;
+
 
 .border {
   border: 2px solid $border-color;
+  border-radius: 20px;
   margin: auto;
-  padding: 20px 30px;
+  padding: 20px 30px 0 30px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `20px 30px 0` instead of `20px 30px 0 30px`